core/lib/basic_types/src/tee_types.rs

```diff
@@ -2,8 +2,57 @@ use serde::{Deserialize, Serialize};
 use strum::{Display, EnumString};
 
 #[derive(Debug, Clone, Copy, PartialEq, EnumString, Display, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+#[strum(serialize_all = "snake_case")]
 #[non_exhaustive]
 pub enum TeeType {
-    #[strum(serialize = "sgx")]
     Sgx,
 }
```

Comment:
Why both `strum` and `serde` tho? I think the real problem here is that the type was _serialized_ with `strum` and deserialized with `serde`.
So wouldn't it be better to just use `serde` to reduce risks of it going out of sync again?

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2798/commits/e524817b6c77f71631b82a453b80c34a2048ed80). ✅